spec/controllers/v2/users/sessions_controller_spec.rb

```diff
@@ -100,7 +106,7 @@
 
           context 'unlock instructions email' do
             it 'sends an instructions email' do
-              expect(UsersMailer.deliveries.size).to eq(1)
```

Comment:
Please don't delete this check. I'd still like to know if all emails have actually been delivered.